prime-router/docs/proposals/0020-universal-metadata-search/example-queries.md

```diff
@@ -0,0 +1,196 @@
+```sql
+select * from delivered_reports_for_senders
+group by created_at;
+```
+- I want a list of reports I sent out sorted by receiver.
+```sql
+select * from delivered_reports_for_senders
+where receiving_org = %receiving_org%;
+```
+- I want a list of reports I sent out grouped by receiver.
+```sql
+select * from delivered_reports_for_senders
+group by receiving_org;
+```
+- I want a list of delivered reports.
+
+**How is this different from reports sorted by data**
+
+- I want a list of undelivered reports.
+
+???
+- I want a list of reports that have warnings.
+
+???
+- I want a list of reports that have errors.
+
+???
+- I want to see the warning and error messages for a given report.
+
+???
+- I want to see what data was sent to whom and when it was sent.
+
+???
+- I want a list of all receivers that received my data.
+```sql
+select distinct receiving_org from delivered_reports_for_senders;
+```
+- I want a list of items that were filtered out for a delivered report.
+???
+
+**1.2** As a RS sender, I want visibility into the data I am sending.
+- I want to see the metadata for a given report.
+    - Ex: See [Submissions](https://staging.reportstream.cdc.gov/submissions) tool on website.
+```sql
+select * from metadata
+where report_id = %report_id%;
+```
+
+**1.3** As a RS receiver, I want visibility into the data I am receiving. See
+[STLT Dashboard Design](https://www.figma.com/proto/6mwI5ac6rprACKDzDo4Ady/ReportStream-Workspace-%7C-2023?node-id=995%3A13227&scaling=min-zoom&page-id=496%3A6448&starting-point-node-id=995%3A13227&show-proto-sidebar=1).
+```sql
+with delivered_reports_for_receiver as (select
+ delivered.*
+from report_file delivered
+join terminal_report_ids on terminal_report_id = delivered.report_id
+join report_file submitted on submitted.report_id = terminal_report_ids.original_report_id
+where delivered.transport_result is not null
+and submitted.sending_org = %receiving_org%)
+```
+
+
+- I want a list of all reports sent to me. See
+  [All available reports](https://www.figma.com/proto/6mwI5ac6rprACKDzDo4Ady/ReportStream-Workspace-%7C-2023?node-id=995%3A13703&scaling=min-zoom&page-id=496%3A6448&starting-point-node-id=995%3A13227&show-proto-sidebar=1).
+```sql
+select * from delivered_reports_for_receiver;
+```
+- I want to filter/group the reports sent to me by the data they contain. See
+  [All available reports](https://www.figma.com/proto/6mwI5ac6rprACKDzDo4Ady/ReportStream-Workspace-%7C-2023?node-id=995%3A13703&scaling=min-zoom&page-id=496%3A6448&starting-point-node-id=995%3A13227&show-proto-sidebar=1).
+    - I want to sort reports sent to me by the date they were sent.
+  
+    ```sql
+    select * from delivered_reports_for_receiver
+    order by created_at
+    ```
+
+    - I want to sort reports sent to me by the date they expire.
+    
+    ???
+    - I want to sort reports sent to me by how many items they contain.
+    ```sql
+    select * from delivered_reports_for_receiver
+    order by item_count;
```

Comment:
May make more sense once you explain above, but wouldn't this need to grouped by a report id?